platforms/documentation/docs/build.gradle.kts

```diff
@@ -508,7 +502,12 @@ samples {
             description = "Pass arguments to a custom task."
             category = "Writing Custom Tasks"
         }
-
+        val customTestTask by creating {
+            sampleDirectory = samplesRoot.dir("writing-tasks/custom-test-task")
+            displayName = "Implementing a task that runs tests"
+            description = "Running tests outside of the JVM."
```

Comment:
💭 "Outside of the JVM" made me expect this would be exec-ing a separate process.  Maybe "outside of a JUnit?" or "with a custom testing framework?"